docs/DEVELOPER_GUIDE.md

```diff
@@ -153,12 +153,12 @@ and deploys the `kfserving-controller-manager` with the image digest to your clu
 
 ### Smoke test after deployment
 ```bash
-kubectl apply -f docs/samples/tensorflow.yaml
+kubectl apply -f docs/samples/tensorflow/tensorflow.yaml
 ```
 You should see model serving deployment running under default or your specified namespace.
 
 ```console
-$ kubectl get pods -n default -l serving.kubeflow.org/kfservice=my-model
+$ kubectl get pods -n default -l serving.kubeflow.org/kfservice=flowers-sample
 NAME                                                READY   STATUS    RESTARTS   AGE
 my-model-default-htz8r-deployment-8fd979f9b-w2qbv   3/3     Running   0          10s
```

Comment:
Can you help change `my-model-default` to `flowers-sample-default` ?

Reply:
Oh... That's great catch! :-)